lib/Decompressor.js

```diff
@@ -36,9 +36,12 @@ export class Decompressor extends Transformer {
    * @param {Map} [options.appContextMap] - A map of JSON-LD Context URLs and
    *   their encoded CBOR-LD values (must be values greater than 32767
    *   (0x7FFF)).
+   * @param {boolean} [options.allowUndefinedTerms] - Allow the JSON-LD document
+   *   to contain terms that are not defined in the context. The original values
+   *   of terms that cannot be resolved from the context map will be preserved.
```

Comment:
Perhaps, good to mention here that: `The key type is expected to be a string for undefined terms`